extensions/amp-story/1.0/page-advancement.js

```diff
@@ -415,6 +421,32 @@ class ManualAdvancement extends AdvancementConfig {
     );
   }
 
+  /**
+   * @param {Event} event
+   * @param {Element} pageEl
+   * @return {boolean}
+   * @private
+   */
+  isWithinInteractiveArea_(event, pageEl) {
+    const pageRect = this.element_.getLayoutBox();
+
+    if (this.isInScreenSideEdge_(event, pageRect)) {
+      return false;
+    }
+
+    if (pageEl.querySelector('amp-story-page-attachment')) {
+      return event.clientY <= pageRect.height - PAGE_ATTACHMENT_SAFETY_AREA_PX;
+    }
+
+    if (pageEl.querySelector('amp-story-cta-layer')) {
+      return (
+        event.clientY <= pageRect.height * (1 - CTA_SAFETY_AREA_PERCENTAGE)
+      );
+    }
+
+    return true;
+  }
```

Comment:
We don't need to worry about the Y position, since the system layer and CTA/page attachment will always be on top of the quiz. You don't need this method and can rely on `this.isInScreenSideEdge_()` alone

Reply:
- The safety area should be the area where the quiz can be positioned as you use `align/justify-self` CSS rules
- If you tap on the left/right edges, we want to make sure we don't block the navigation, hence the `isInScreenSideEdge_`
- If you tap on the top/bottom edges, there's no action that could be blocked as the CTA/System layer buttons will always be on top, so we don't need this code

What we need instead though, is to change the default padding-bottom of the amp-story-grid-layer when there's a CTA or a page attachment

The correct behavior should be:

1. If a publisher positions the quiz with `align/justify-self` it can never go on the left/right edges to block navigation, and can never overlap with the cta/system layer. For that to happen you will have to change the padding-bottom. You should be able to do it without JavaScript, using the general sibling selector `~`
2. If a publisher decides to go crazy and use `position: absolute;` they can, but the current `z-index` configuration ensures that the quiz will always be under the CTA or system layer, and the `this.isInScreenSideEdge_()` ensures that it won't block navigation

Reply:
Changing grid layer's padding bottom when there's a cta layer sounds like it would have been the right idea, but at this point, isn't that a breaking change?